### ISSUE TITLE
Editor: Posts: Each autosave saves as unique draft post

### ISSUE THREAD

**Issue by courane01** [Automattic/wp-calypso#1376]
Each time a post autosaves in the app, my self-hosted Jetpack enabled wp-admin dashboard shows a new post as draft.  Additionally, autosaves brought my view from wherever I was typing within a long post to the top of the post in the WordPress.com app.

**Comment by lancewillett** [comment]
Thanks for the reports courane01

> 1. Each time a post autosaves in the app, my self-hosted Jetpack enabled wp-admin dashboard shows a new post as draft.

This sounds like unexpected behavior, right? Do you mean that each time autosave happens in Calypso (sounds like you're using one of the desktop apps) you get a **new** draft created in your wp-admin posts list? Could you upload a screenshot of that post list, if you don't mind?

I can't repeat that, using the Mac WordPress.com app and a self-hosted Jetpack install running both trunk WordPress and 3.8.1.

Could you please share the following with us?
- The WordPress version in the self-hosted Jetpack install
- The Jetpack plugin version

> 2. Additionally, autosaves brought my view from wherever I was typing within a long post to the top of the post in the WordPress.com app.

I couldn't repeat this one. Are you using the Mac or Windows desktop app? Which OS version are you on?

**Comment by courane01** [comment]
I am running WordPress 4.4 and Jetpack 3.8.1. As I wrote this post close to the 4.4 rollout, it may have been the earlier WP version.  I am on OX X El Capitan and the most current version of the WordPress.com app.  I host with Siteground if that matters.  I keep everything I have fairly up to date, but am still on php 5.6.x? Whatever was before PHP7 as I still have a conflict somewhere w/ PHP7.  

I've already moved all duplicate posts to my trash, but based upon the post I wrote with the app, the slug would indicate I had 50 versions of this in the roughly 90mins of writing time.  

http://courtneyengle.com/2015/12/08/wordcamp-us-friends-50/ 

![screen shot 2015-12-15 at 5 16 18 pm](https://cloud.githubusercontent.com/assets/5519705/11825889/ec514c1e-a34f-11e5-9641-65737f54f80a.png)

I would be happy to use my Siteground staging server and grant access to this for any other efforts.
